### ISSUE TITLE
NetAnalyzers 7 packages are still preview

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn-analyzers#6285]
.NET 7 stable was already released. I think a stable NetAnalyzers 7 package should be released.

cc @jmarolf @mavasani

**Comment by mavasani** [comment]
Also tagging @arkalyanms

**Comment by Youssef1313** [comment]
@mavasani Thanks!
Does this reflect in dotnet/sdk automatically or needs a manual change to `release/7.0.1xx` branch?

**Comment by jmarolf** [comment]
> Does this reflect in dotnet/sdk automatically or needs a manual change to release/7.0.1xx branch?

The release packages were alreaady shipped in the SDK, we just forget to publish the associated release packages to nuget.org. The published package corresponds to the SHA of what is currently in the .NET 7.0.100 SDK

**Comment by cremor** [comment]
> The published package corresponds to the SHA of what is currently in the .NET 7.0.100 SDK

@jmarolf If that is the case, then why can I still reproduce #6141 with SDK 7.0.102, but installing the NuGet package "Microsoft.CodeAnalysis.NetAnalyzers" 7.0.0 manually fixes it?

**Comment by dotnetshadow** [comment]
I agree with @cremor I'm still facing this issue with the latest sdk 7.0.202

**Comment by VictorioBerra** [comment]
Still an issue with 7.0.203
